evaluator/diagnostic_converter/CMakeLists.txt

```diff
@@ -0,0 +1,38 @@
+cmake_minimum_required(VERSION 3.16.3) # Ubuntu 20.04 default CMake version
+
+project(diagnostic_converter)
+
+if(NOT CMAKE_CXX_STANDARD)
+  set(CMAKE_CXX_STANDARD 14)
+endif()
+
+if(CMAKE_COMPILER_IS_GNUCXX OR CMAKE_CXX_COMPILER_ID MATCHES "Clang")
+  add_compile_options(-Wall -Wextra -Wpedantic)
+endif()
+
+find_package(ament_cmake_auto REQUIRED)
+
+ament_auto_find_build_dependencies()
+
+ament_auto_add_library(${PROJECT_NAME}_node SHARED
+  src/converter_node.cpp
+)
+
+rclcpp_components_register_node(${PROJECT_NAME}_node
```

Comment:
You may need to write a dependency on `pluginlib` to use this macro.

https://github.com/autowarefoundation/autoware.universe/pull/2973

Reply:
@KeisukeShima 
Thank you for the comment!
The change I made is appropriate?